recipes/semi-ate-sammy/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "semi-ate-sammy" %}
+{% set version = "1.0.6" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/semi-ate-sammy-{{ version }}.tar.gz
+  sha256: 2bbc3f5b7aad012902a48656168788111b21ee4b3b2f9f177299838e2f016234
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - sammy=ate_sammy.sammy:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+  run:
+    - jinja2
+    - python >=3.8
+    - semi-ate-project-database ==1.0.6
```

If this will always be kept in sync:
```suggestion
    - semi-ate-project-database =={{ version }}
```